tests/unit/test-util/test.adb.js

```diff
@@ -1129,6 +1131,44 @@ describe('utils/adb', () => {
       );
     });
 
+    it('informs the user if remote debugging is not enabled', async () => {
+      const adb = getFakeADBKit({
+        adbClient: {
+          shell: sinon.spy(() => Promise.resolve('')),
+        },
+        adbkitUtil: {
+          readAll: sinon.spy(() => {
+            return Promise.resolve(Buffer.from(''));
+          }),
+        },
+      });
+      const adbUtils = new ADBUtils({adb});
+
+      const maxDiscoveryTime = 50;
+      const retryInterval = 10;
+
+      consoleStream.stopCapturing();
+      consoleStream.flushCapturedLogs();
+      consoleStream.startCapturing();
+      consoleStream.makeVerbose();
```

Comment:
I think that the following tweaked version should be enough and slightly better:
```
      consoleStream.flushCapturedLogs(); <--- clear the captured logs
      consoleStream.makeVerbose(); <--- make the log verbose (to be able to assert the log level)
      consoleStream.startCapturing(); <--- make sure that we start capturing logs
      ...
      const promise = ... <-- get the promise
      await assert.isRejected(promise, WebExtError);  <--- wait the async function to complete
      ... // get the message that we expect by filtering the captured logs
      consoleStream.stopCapturing(); <--- stop capturing we don't need it anymore
      ... // assertions on the expected message and log level
```